docs/en/operation/Metrics-System.md

```diff
@@ -120,9 +125,208 @@ Metrics in Alluxio have the following format for non-master node metrics:
 
 [processType].[hostName].[metricName].[tag1].[tag2]...
 
-The list of process metrics exposed by the master or workers can be found at the `/metrics/json/`
-endpoint of the web UI. There is generally an Alluxio metric for every RPC invocation, to Alluxio or
-to the under store.
+There is generally an Alluxio metric for every RPC invocation, to Alluxio or to the under store.
 
 Tags are additional pieces of metadata for the metric such as user name or under storage location.
 Tags can be used to further filter or aggregate on various characteristics.
+
+### Cluster Metrics
+
+![Master Metrics]({{ '/img/screenshot_generalMetrics.png' | relativize_url }})
+
+Workers and clients send metrics data to the Alluxio master through heartbeats.
+The interval is defined by property `alluxio.master.worker.heartbeat.interval` and `alluxio.user.metrics.heartbeat.interval` respectively.
+
+Each client will be assigned an application id. 
+All the metrics sent by this client contain the client application id information. 
+By default, this will be in the form of 'app-[random number]'. 
+This value can be configured through the property `alluxio.user.app.id`, 
+so multiple clients can be combined into a logical application.
+
+* Alluxio cluster information
```

Comment:
Maybe for a different PR - we should think of how to do this programmatically, it is hard to keep the code and docs in sync.

Reply:
Agreed that we should make it programmatic. Some metrics are auto-generated, like those RPC metrics and Failure metrics, and may be hard to make it programmatically. It will be better to do it in a separate PR